mcp/cloudflare-worker/worker.ts

```
@@ -0,0 +1,522 @@
+            await this.makeRequest(`/workspaces/${this.config.workspaceId}/sessions`, {
+                method: 'POST',
+                body: JSON.stringify({
+                    id: sessionId,
+                }),
+            });
+
+            // Add peers to the session
+            await this.makeRequest(`/workspaces/${this.config.workspaceId}/sessions/${sessionId}/peers`, {
+                method: 'POST',
+                body: JSON.stringify({
+                    [this.config.userName!]: {},
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)